src/extension.ts

```diff
@@ -62,10 +62,13 @@ export function activate(context: ExtensionContext) {
 
     // Register support for unicode input.
     (async () => {
-        const translations: any = await loadJsonFile.default(context.asAbsolutePath('translations.json'));
+        const translations: any = await loadJsonFile(context.asAbsolutePath('translations.json'));
+        const inputLanguages: string[] = workspace.getConfiguration('lean.input').get('languages', ['lean']);
```

Comment:
Please use the `inputModeLanguages()` function you added to `input.ts`.  (You need to add the `export` modifier to use it from other files.)